Formula/squashfs.rb

```diff
@@ -25,7 +28,7 @@ class Squashfs < Formula
 
   def install
     cd "squashfs-tools" do
-      system "make XATTR_SUPPORT=0 EXTRA_CFLAGS=-std=gnu89 LZO_SUPPORT=1 LZO_DIR='#{HOMEBREW_PREFIX}' XZ_SUPPORT=1 XZ_DIR='#{HOMEBREW_PREFIX}' LZMA_XZ_SUPPORT=1"
+      system "make XATTR_SUPPORT=0 EXTRA_CFLAGS=-std=gnu89 LZO_SUPPORT=1 LZO_DIR='#{HOMEBREW_PREFIX}' XZ_SUPPORT=1 XZ_DIR='#{HOMEBREW_PREFIX}' LZMA_XZ_SUPPORT=1" + (build.with? "lz4" ? " LZ4_SUPPORT=1" : "")
```

Comment:
A better way to do this is to create an `args` array then conditionally add the lz4 option:
```ruby
args %W[
  XATTR_SUPPORT=0
  EXTRA_CFLAGS=-std=gnu89
  LZO_SUPPORT=1
  LZO_DIR=#{HOMEBREW_PREFIX}
  etc...
]
args << "LZ4_SUPPORT=1" if build.with? "lz4"

system "make", *args